example/python/README.md

```diff
@@ -0,0 +1,33 @@
+# Python client library example
+
+## Prerequisites
+
+In order to execute script demonstrating execution of python client library you need to have python 2.7 installed. After that please follow next steps:
```

Comment:
I believe everything should work with p3 too. At least in theory :D

Reply:
I tested using python 2. Need check if pyhton 3 also suitable

Reply:
No, probably add 3rd python support in next releases

Reply:
Actually it works pretty fine

Reply:
wow